kitty/cli.py

```diff
@@ -47,6 +47,10 @@
 Run python code in the kitty context
 
 
+--remote -r
+Run command in running instance
```

Comment:
The help message should be expanded to explain that it works via the --instance-group option that must be specified as well.